source/_integrations/zwave.markdown

```diff
@@ -20,6 +20,12 @@ ha_codeowners:
 ha_domain: zwave
 ---
 
+<p class='note warning'>
+This integration is deprecated. We recommend using <a href="/integrations/zwave_js">the Z-Wave JS integration</a>.
+<br><br>
+It will no longer receive any updates. It will not be removed unless it becomes incompatible with a future version of Python.
```

Comment:
This is grammatically referring to the Z-Wave JS integration, but that's not what we really mean.

Reply:
Good point

Reply:
```suggestion
The Z-Wave integration will no longer receive any updates. It will not be removed unless it becomes incompatible with a future version of Python.
```